common/autoware_trajectory/src/detail/types.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright 2024 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "autoware/trajectory/detail/types.hpp"
+
+namespace autoware::trajectory::detail
+{
+MutablePoint3d to_point(geometry_msgs::msg::Point & p)
```

Comment:
add const to the argument and function itself.

Reply:
This function is for creating non-const reference, so this is OK